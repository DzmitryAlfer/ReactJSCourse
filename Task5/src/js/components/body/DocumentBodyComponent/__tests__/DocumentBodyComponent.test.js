import React from 'react';
import Renderer from 'react-test-renderer';
import {DocumentBodyComponent} from "../";

jest.mock('../../search/SearchResultsContainer/index');

describe('DocumentBodyComponent', () => {
    test('Component render', () => {
        const tree = Renderer.create(
            <DocumentBodyComponent/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});