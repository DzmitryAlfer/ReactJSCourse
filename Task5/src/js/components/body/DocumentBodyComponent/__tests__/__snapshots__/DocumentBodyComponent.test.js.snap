// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentBodyComponent Component render 1`] = `
<main
  className="main-body"
>
  <span
    onItemClick={undefined}
    searchString={undefined}
  >
    SearchResultsContainer MOCK
  </span>
</main>
`;
